pkg/nsx/services/vpc/builder.go

```diff
@@ -27,6 +30,19 @@ func generateIPBlockKey(block model.IpAddressBlock) string {
 	return *cidr + "_" + nsUID
 }
 
+func generateLBSKey(lbs model.LBService) string {
+	if lbs.ConnectivityPath == nil {
+		return ""
```

Comment:
which scenario the ConnectivityPath is a nil?  needs to log a error for LBService?